.github/workflows/tests.yml

```diff
@@ -12,8 +12,21 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.22.2'
+          go-version: "1.22.2"
           cache-dependency-path: "**/go.sum"
       - name: Run the tests
         run: make run-tests
         working-directory: ./backend
+
+  frontend-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
+      - run: npm install
+        working-directory: ./frontend
+      - run: npm test
```

Comment:
Could we use the `make` command here, for consistency?

For that, we should add a variable that controls whether we want to run tests in watch mode or not in the Makefile.

Reply:
I will check it now 👌